src/javascript/app/SearchView/SearchView.jsx

```diff
@@ -64,7 +64,7 @@ const buildAndFireSearchEvent = searchTerm => {
 
     searchEvent.properties = {
         originForm: 'augmentedSearchForm',
-        language: window.contextJsParameters.lang,
+        language: window.digitalData.page.pageInfo.language,
```

Comment:
As a global object we should ensure that `digitalData` is set, it will generate an error without it:
```window.digitalData?.page.pageInfo.language```

Reply:
done